apps/rehabstod/src/store/api.ts

```diff
@@ -24,6 +24,10 @@ export const api = createApi({
       query: () => 'user',
       providesTags: ['User'],
     }),
+    getConfig: builder.query<Config, void>({
+      query: () => 'config',
+      providesTags: ['User'],
```

Comment:
Why does this query provide the tag 'User'?

Reply:
Copy paste error :)